src/storm-pomdp/parser/PomdpSolveParser.h

```diff
@@ -0,0 +1,66 @@
+#pragma once
+#include <string>
+
+namespace storm {
+namespace models {
+namespace sparse {
+// Forward declaration
+template<typename ValueType, typename RewardModelType>
+class Pomdp;
+template<typename ValueType>
+class StandardRewardModel;
+}  // namespace sparse
+}  // namespace models
+namespace pomdp {
+namespace parser {
+
+struct PomdpData {
```

Comment:
This seems to be an unused duplicate of the struct in the .cpp. I would keep the one in the header.